### ISSUE TITLE
Use additionnal font

### ISSUE THREAD

**Issue by Almtesh** [pwaller/pyfiglet#51]
Hi,
I want to use a additionnal font that is not in the fonts directory.
I use this as a Python3 library. I used the font param from Figlet class, but it does not work with path and openned file.

**Comment by pwaller** [comment]
Please provide complete instructions and materials for reproducing.

**Comment by Almtesh** [comment]
Debian 8.7 package python3-pyfiglet 0.7.2+dfsg-1
```
>>> from pyfiglet import Figlet
>>> f = Figlet (font = '/home/almtesh/doom.flf')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3/dist-packages/pyfiglet/__init__.py", line 527, in __init__
    self.setFont()
  File "/usr/lib/python3/dist-packages/pyfiglet/__init__.py", line 534, in setFont
    self.Font = FigletFont(font=self.font)
  File "/usr/lib/python3/dist-packages/pyfiglet/__init__.py", line 85, in __init__
    self.data = self.preloadFont(font)
  File "/usr/lib/python3/dist-packages/pyfiglet/__init__.py", line 100, in preloadFont
    raise FontNotFound(font)
pyfiglet.FontNotFound: /home/almtesh/doom.flf
```

**Comment by irskep** [comment]
I fixed this in #53.

**Comment by Almtesh** [comment]
Thanks a lot. I'll try this when I'll have some time to do.
